### ISSUE TITLE
Update OAGIS Model Generation Code to be able to generate releases

### ISSUE THREAD

**Issue by kbserm** [OAGi/Score#465]
Also add a UI button for this generation, maybe on the Release Management page.  

One question is how do we deliver the finished model. I'm thinking to store it as a zip BLOB in the DB. So once the generation is done, a download link is added to the release management page. 

How can we notify the user when the generation is done? I'm okay with the simple method, to keep refreshing the page until the download link appear, but what happen if the generation fail. 

Should we need to use a queue?

The user should be able to generate model XML Schemas for draft releases as well. When the draft releases are deleted, those blobs should be deleted too.

**Comment by kbserm** [comment]
since this is run in the background process and we won't do much of the Score-based validation, we could run the whole generated schemas thru schema validation, if we have time. Otherwise, in the user guide we tell the user to validate generated schemas off-line for now.

**Comment by kbserm** [comment]
Code is available but UI is not ready b/c of Module Management. Move this to 2.x for the record.
